gt7dashboard/test/test_gt7communication.py

```diff
@@ -1,16 +1,18 @@
+import logging
 import os
 import time
 import unittest
 
 from gt7dashboard import gt7communication
 from gt7dashboard.gt7lap import Lap
 
-PLAYSTATION_IP = "ps5wifi"
+PLAYSTATION_IP = "192.168.15.96"
```

Comment:
Maybe we should find a solution that works for both of us. In the mean time, please do not change the IP.